frontend/packages/helm-plugin/src/catalog/components/HelmCatalogTypeDescription.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { QuickStartContextValues, QuickStartContext } from '@patternfly/quickstarts';
+import { Trans } from 'react-i18next';
+import { Link, useLocation } from 'react-router-dom';
+import { isModifiedEvent } from '@console/shared/src';
+
+const HelmCatalogTypeDescription: React.FC = () => {
+  const { setActiveQuickStart } = React.useContext<QuickStartContextValues>(QuickStartContext);
+  const { pathname, search } = useLocation();
+  const queryParams = new URLSearchParams(search);
+  queryParams.set('quickstart', 'install-helmchartrepo-ns');
+
+  const to = {
+    pathname,
+    search: `?${queryParams.toString()}`,
+  };
+
+  const handleOnClick = (event: React.MouseEvent<HTMLElement>) => {
+    if (isModifiedEvent(event)) {
+      event.preventDefault();
```

Comment:
When detecting a ctrl+click we should not start the quick start in the current tab, or? Wdyt?

```suggestion
      event.preventDefault();
      return;
```